jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/sftp/AutoConfiguration.java

```diff
@@ -87,20 +88,23 @@ public AutoConfiguration(SftpInputProperties sftpInputProperties) {
     public SftpRemoteFileTemplate template() {
         try {
             return new SftpRemoteFileTemplate(sftpSessionFactory());
-        }catch(InvalidConfigException ex)
-        {
+        } catch (InvalidConfigException ex) {
             logger.error(ex.getMessage());
         }
         return null;
     }
 
     @Bean
     @InboundChannelAdapter(channel = "sftpChannel", poller = @Poller(cron = "${bcgov.access.input.sftp.cron}", maxMessagesPerPoll = "${bcgov.access.input.sftp.max-message-per-poll}"))
-    public MessageSource<InputStream> sftpMessageSource() {
+    public MessageSource<InputStream> sftpMessageSource(RedisMetadataStore redisMetadataStore) {
```

Comment:
question: is it possible to add a unit test for this method?

Reply:
@alexjoybc 
yes. it is working. Change to ConcurrentMetadataStore.

Reply:
@sdevalapurkar , you mean add test for "AutoConfiguration" class. Sure, I will do that.